lint/lint_test.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lint
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/desc/builder"
+)
+
+func TestLinter_run(t *testing.T) {
+	fd, err := builder.NewFile("protofile.proto").Build()
+	if err != nil {
+		t.Fatalf("Failed to build a file descriptor.")
+	}
+	defaultConfigs := Configs{
+		{[]string{"**"}, []string{}, map[string]RuleConfig{}},
```

Comment:
I think it is more informative to write down the field name, e.g., `IncludedPaths: []string{"**"}`, which also prevents issues when we refactor the `Config` struct, e.g., by inserting a new field.

Reply:
This code is not actually changed (it is only in the diff because I renamed the file).

Reply:
Got it -- it is a nit pick -- we can refactor it later. Thanks!